packages/legend-application/style/components/_virtual-assistant.scss

```diff
@@ -299,6 +299,35 @@
     position: relative;
   }
 
+  &__guide {
```

Comment:
Let's componentize this

Reply:
ye makes sense! added new .scss file for text_search